tests/unittest.py

```diff
@@ -14,21 +14,30 @@
 # limitations under the License.
 import twisted
 from twisted.trial import unittest
+from synapse.util.logcontext import LoggingContextFilter
```

Comment:
ftr, PEP8 has some [opinions](https://www.python.org/dev/peps/pep-0008/#imports) on import order which we try to follow - in this case that means that imports from `synapse` go after those from `twisted`. OTOH this file already breaks the convention by putting `import logging` at the end so \<shrug>